src/app/modules/customer-management/components/projects-type/components/create-project-type/create-project-type.component.spec.ts

```diff
@@ -121,6 +122,13 @@ describe('InputProjectTypeComponent', () => {
     expect(store.dispatch).toHaveBeenCalledWith(new CreateProjectType(projectTypeData));
   });
 
+  it('should reset Project Type Form if projectTypeIdToEdit is null', () => {
+    spyOn(component.projectTypeForm, 'reset');
+    projectTypeIdToEditMock = store.overrideSelector(projectTypeIdToEdit, null);
```

Comment:
what is this variable "projectTypeIdToEditMock" used for?

Reply:
It is not necessary I will remove it